### DISCUSSION TITLE
Linux support?

### DISCUSSION THREAD

**Discussion by PunyTRexArms** [brackets-cont/brackets#246] [General]
Many thanks to the community of developers that continue to make Brackets a success.

A year ago there was a inquiry as to a possible Linux port of Brackets. Is this still an objective, and if so, is there a time line for completion?

**Comment by abose** [comment]
Started on it mid December 2022: https://github.com/brackets-cont/brackets/issues/112
See https://github.com/orgs/phcode-dev/projects/1 to track the progress

**Comment by charlypa** [comment]
@PunyTRexArms 

We've updated our editor to Phoenix Code, launched just yesterday. It now supports Windows, Mac, and Linux moving beyond the older Brackets x86 binaries.
You can download Phoenix Code at https://phcode.io/.
